wgms-workgroups/src/test/java/com/sailpoint/atlas/wgms/workgroups/integration/BaseIntegrationTest.java

```
@@ -0,0 +1,344 @@
+		return new HashMap<String, Object>() {{
+			put(JOIN_OPERATOR, joinOp);
+			put(FILTER, filters);
+		}};
+	}
+
+	protected Map<String, String> buildSorter(String property, String direction) {
+		return new HashMap<String, String>() {{
+			put(PROPERTY, property);
+			put(DIRECTION, direction);
+		}};
+	}
+
+	/**
+	 * Sets a value in the params if the value is not null.
+	 *
+	 * @param params The http request parameter object.
+	 * @param key The parameter name.
+	 * @param value Value of the parameter.
+	 */
+	private void setIfNotNull(Params params, String key, Object value) {
+		if(value != null) {
+			params.query(key, value);
+		}
+	}
+
+	/**
+	 * Gets a rest client for the given pod and org.
+	 *
+	 * @param pod The pod.
+	 * @param org The org.
+	 * @return The Rest API client.
+	 */
+	protected BaseRestClient getRestClient(String pod, String org) {
+
+		try {
+			_application.getRestPort().ifPresent((restPort) -> {
+				wgmsRestClient = new IdnAtlasClient(
+						String.format(
+								"http://localhost:%d",
+								new Object[]{Integer.valueOf(restPort)}
+						),
+						pod,
+						org,
+						pod + "#" + org,
+						(String) null,
+						this.getHttpClient()
+				);
+			});
+		} catch (Throwable var2) {
+			var2.printStackTrace();
+			stopApplication();
+			throw var2;
+		}
+		return wgmsRestClient;
+	}
+
+	/**
+	 * Instantiates a JDBC connection, creates a statement and sends it to the executor.
+	 * @param executor The executor to run.
+	 */
+	public void execute(StatementExecutor executor) {
+		Connection connection = null;
+		Statement statement = null;
+
+		try {
+			connection = getJdbcConnection();
+			statement = connection.createStatement();
+			executor.execute(statement);
+		} catch (Exception ex) {
+			throw new RuntimeException("error executing statement", ex);
+		} finally {
+			if (statement != null) {
+				try { statement.close(); } catch (Exception ignored) { }
```

Put something in empty catch blocks.

---

Addressed